app/services/provider_authorisation.rb

```diff
@@ -48,6 +48,12 @@ def can_view_safeguarding_information?(course:)
     end
   end
 
+  def can_manage_organisation?(provider:)
+    return true if @actor.is_a?(SupportUser)
+
+    @actor.provider_permissions.exists?(provider: provider, manage_organisations: true)
```

Comment:
Actor may also be a VendorApiUser. This model responds to .providers but not `.provider_permissions'. Not particularly important as api keys cannot currently manage org relationships, but worth a mention.